lottie-ios/Classes/AnimatableProperties/LOTAnimatableBoundsValue.m

```diff
@@ -182,7 +182,7 @@ - (nullable CAKeyframeAnimation *)animationForKeyPath:(nonnull NSString *)keypat
   keyframeAnimation.duration = self.duration;
   keyframeAnimation.beginTime = self.delay;
   keyframeAnimation.fillMode = kCAFillModeForwards;
-  return keyframeAnimation;
+  return @[keyframeAnimation];
```

Comment:
@buba447 Thanks for taking a look at this!

I've updated the signature of the `LOTAnimatableValue` method for querying animations to be able to return more than one animation (with no animation is now represented by an empty array).

This behavior is required to ensure that `LOTAnimatableScaleValue` returns more than one `CAKeyframeAnimation`: one to animate `transform.scale.x` and another to animate `transform.scale.y`. Whereas previously a single animation with a keypath of `transform` was returned, we now only animate a single property of the `transform` (the scale). This allows for the rotation to be animated separately via the `transform.rotation.z` keypath without clobbering the transform.scale`.

Hope that makes sense—let me know if anything needs additional clarification!